docs/user-journeys/users/application-developer/intermediate.md

```diff
@@ -10,68 +10,102 @@ track: "USERS › APPLICATION DEVELOPER › INTERMEDIATE"
+https://docs.google.com/a/heptio.com/spreadsheets/d/1FCgqz1Ci7_VCz_wdh8vBitZ3giBtac_H8SBw4uxnrsE/edit?usp=drive_web
 
-Aenean suscipit arcu ac leo tincidunt tempus. Donec maximus tellus libero, ac ullamcorper magna lobortis ac. Integer mollis nisl vitae magna gravida, nec ornare ex consectetur. Sed mattis tincidunt nisi, at consequat tellus malesuada non. Integer vel semper nisi, ut fringilla velit. Nam felis ex, congue non dui vitae, sollicitudin convallis turpis. Phasellus porttitor maximus turpis, in varius nibh fermentum aliquam. Cras finibus lacus non diam porttitor porttitor. Nulla fringilla sagittis nibh nec condimentum. Duis egestas mauris nec dolor hendrerit ullamcorper sit amet in mi. Phasellus sollicitudin justo diam.
+## Beyond Deployments: More workload patterns
 
-## Work with commonly used features
+As you gain familiarity with Kubernetes and your use cases become more complex, you may find it helpful to familiarize yourself with more of the toolkit that Kubernetes provides (beyond the [common Kubernetes object types](/docs/user-journeys/users/application-developer/foundational/#section-2){:target="_blank"}).
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean id felis non enim faucibus lacinia. Aliquam massa mauris, interdum a ex ut, sagittis rutrum nulla. In pellentesque est at molestie fringilla. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Donec eu mi at velit lacinia venenatis ac nec sem. In volutpat pellentesque dui ut commodo. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Duis sollicitudin eleifend felis non facilisis. Pellentesque leo urna, congue id auctor non, varius a nunc. Duis ultrices, odio ut hendrerit suscipit, nisi mauris dignissim mauris, nec bibendum ante neque ut augue. Interdum et malesuada fames ac ante ipsum primis in faucibus. Nam condimentum libero sit amet rutrum fermentum. Proin posuere condimentum odio. Aenean volutpat, ex vel tincidunt tincidunt, massa odio condimentum lectus, vel iaculis libero dolor in nisi.
+#### Workloads
 
-Curabitur bibendum tempor mi, vel lacinia nisi vulputate ac. Nulla dignissim consectetur nisl nec tincidunt. Etiam pharetra facilisis sapien, non gravida velit fermentum sed. Ut ac ultrices nunc, in vestibulum urna. Suspendisse accumsan euismod felis, sit amet rhoncus neque volutpat luctus. Aliquam tincidunt pellentesque mauris, sed tempus diam. Mauris in elit eget justo tempor suscipit.
+{% glossary_tooltip text="Deployments" term_id="deployment" %}, which you previously covered, are ideal for running, updating, and scaling stateless web servers (e.g. a Rails app). However, for other workloads, you may find other Kubernetes API resources more useful:
```

Comment:
I think there's a fundamental distinction missing here. Deployments are good for for both stateless and stateful apps, as we demonstrated with PersistentVolumes in the beginner section. So we need to say what makes StatefulSets even more stateful than what you get with PersistentVolumes.

I'll revisit this comment and make a concrete suggestion.